Sources/MapboxCoreNavigation/Distance/GantryDistanceInfo.swift

```diff
@@ -9,6 +9,9 @@ public struct GantryDistanceInfo {
     /** Distance to the gantry object */
     public let distance: CLLocationDistance
 
+    /**
+     Initializes a new `GantryDistanceInfo` object with a given distance to the object.
```

Comment:
I think the second “object” probably refers to a road object, as opposed to a Swift object. This might be less confusing:

```suggestion
     Initializes a new `GantryDistanceInfo` object with a given distance to the gantry.
```

Reply:
I think using `road object` might be less confusing.